### PR TITLE
Clarify the problem and its temp resolution

### PR DESCRIPTION
**Pull request by janicki**
The problem affects receiving email, not just sending, so that was clarified, among other things.  Also, a typo (SPN->APN) was fixed.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-71b6cdf252/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-24a36b6f4e/how-to/).

**Comment by DonNateR** [comment]
@janicki: Could you describe how you confirmed that this was an issue specific only for T-Mobile Andriod phones?

**Comment by janicki** [comment]
Regarding your T-Mobile question, I didn't actually confirm it was *only* T-Mobile, but it definitely included T-Mobile.  I don't remember what I wrote, but sorry that was unclear.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-64c617623e/how-to/)

File: content/rackspace-email/email-issue-on-tmobile-android-phones.md [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-last_modified_date: '2015-09-15'
-last_modified_by: Kelly Holcomb
+last_modified_date: '2018-01-14'
+last_modified_by: Nate Archer
 product: Rackspace Email
 product_url: rackspace-email
 ---
 
-If you are an Android mobile phone user, you might sometimes experience problems sending outgoing email, and attempts to reconnect to the outgoing server result in the following error message:
+If you are an Android mobile phone user, you might sometimes experience problems sending or receiving email.  Attempts to receive email might be faster than normal (i.e. when a connection is successful, but no new email is found).  Attempts to connect to the outgoing server might fail without error or result in the following error message:
```
Comment (cyrichardson) [inline content/rackspace-email/email-issue-on-tmobile-android-phones.md]: ...receiving email. Attempts to receive email might be faster than normal (such as when a connection is successful, but no new email is found).

(There was an extra space between "email." and "Attempts. Also, changed "i.e" to "such as")

File: content/rackspace-email/email-issue-on-tmobile-android-phones.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+The procedure in this article was created by the Rackspace Technical Community team as a temporary solution to this issue. A more permamnent solution is being investigated.
 
 **Note:** The procedure might be different for each Android mobile phone. The instructions in this article were created using a Galaxy Note 3.
 
 ### Access Point Name
 
 The Access Point Name (APN) is a gateway between a GPRS, 3G, or 4G mobile network and another computer network that runs on many Android devices. The outgoing email problem on Android phones occurs because the APN, which for T-Mobile is `fast.t-mobile.com`, is set to use IPv6 only and does not allow the outbound email server to respond. This in turn keeps emails on Android phones in the outbox until it connects to a Wi-Fi network. The default APN doesn't roll back to IPv4, so the SMTP connection cannot be made. The way to fix this is to add a new APN that does roll back to IPv4.
 
+Note that you might experience no problems using the default APN for other uses, such as browsing websites or using network-connected apps.
```
Comment (cyrichardson) [inline content/rackspace-email/email-issue-on-tmobile-android-phones.md]: when accessing the default APN...

File: content/rackspace-email/email-issue-on-tmobile-android-phones.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
-title: Outgoing email issue on Android phones
+permalink: email-issue-on-android-phones/
+audit_date: '2018-01-14'
+title: Email issue on Android phones
 type: article
 created_date: '2015-03-31'
 created_by: Rose Contreras
-last_modified_date: '2015-09-15'
-last_modified_by: Kelly Holcomb
+last_modified_date: '2018-01-14'
```
Comment (cyrichardson) [inline content/rackspace-email/email-issue-on-tmobile-android-phones.md]: Does this need to be today's date?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a88ae040d6/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**